tests/e2e/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "googlesitekit-e2e",
+  "description": "E2E testing infrastructure for Google Site Kit",
+  "main": "script.js",
+  "private": true,
```

Comment:
This is not needed, we can remove it.
```suggestion
```

Reply:
Actually after working @aaemnnosttv to install npm 7 as a dev dependency we do need these fields to be able to run the command:

```
  npx npm install -w tests/e2e
```

We might not need this once the main npm version support workspaces.

You can see the commit that supports this here: https://github.com/google/site-kit-wp/pull/10119/commits/2912b484b62388626d30103b5352cae12ce5bbeb